content/en/notebooks/advanced_analysis/_index.md

```diff
@@ -0,0 +1,122 @@
+| Operation | Description |
+|-----------|-------------|
+| Parse | Enter [grok syntax][2] to extract data into a separate column. In the "from" dropdown menu, select the column the data is getting extracted from. |
+| Group | Select what you want to group the data by in the dropdown menus. |
+| Join | Select the type of join, the dataset to join against, and the fields to join on. |
+| Filter | Add a filter query for the dataset. |
+| Calculate | Add a name for the field and the function formula, using the [calculated field expression language][3]. |
+| Limit | Enter the number of rows of the dataset you want to display. |
+| Sort | Select the sort order and column to sort on. |
+| Convert | Select the column and the column type to be converted. |
+
+### Analysis cell
+
+You can also transform your data using SQL by adding an analysis cell to your notebook.
+
+1. Type `/sql` or `/analysis` and press <kbd>Enter</kbd> or click the **Analyze Dataset** tile at the bottom of the page.   
+2. In the source dataset dropdown, select the data source you want to transform.  
+3. Write your SQL query. For supported SQL syntax, see the [DDSQL Reference][4].  
+4. Click **Run** in the top-right corner of the analysis cell to execute your query.
+
+## Viewing and exporting data
+
+For any computational cell that includes a dataset preview, you can view the full 100-row preview by clicking the **View dataset** button.
+
+You can save the results of any computational cell to a Dashboard by clicking **Save to Dashboard** and selecting an existing dashboard or creating a new one. Although this creates a sync between your notebook cell and the exported dashboard graph, changes to the query in your notebook do not automatically update the dashboard. 
+
+If you update the published cell or any upstream cells, a badge appears in the upper-right corner of the cell indicating **unpublished changes**. After you publish those changes, the updates sync to **all** dashboards where the query is used.
+
+### Download dataset as a CSV
+
+You can download the data produced by your queries in CSV format for use outside of Datadog. On any computational cell, click the download icon and choose the number of rows to export. 
+
+## Visualizing transformed data
+
+You can graph the data you've transformed using computational cells inside a notebook, customizing the visualization with filters, aggregations, and appearance settings.
+
+To graph your data:
+
+1. Type `/graph` and press <kbd>Enter</kbd> or click the **graph dataset** tile at the bottom of the page.  
+2. Type or select your desired data source from the drop down menu and press <kbd>Enter</kbd>.  
+3. Select your visualization type from the graph menu and press <kbd>Enter</kbd>.
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/logs
+[2]: /logs/log_configuration/parsing/
+[3]: /logs/explorer/calculated_fields/expression_language/
+[4]: /ddsql_reference/
```

Comment:
My last comment is that we should add an examples section, we can just copy the content of the log workspace documentation: https://docs.datadoghq.com/logs/workspaces/

Reply:
I copied over the Examples into a separate Getting Started subpage